Get grid of eval-rst from quickstart doc (#1330)
poliastro/poliastro: docs/source/quickstart.md

  '2000-01-01 12:00:00.000'
  ```

* The reference frame of the orbit will be one pseudo-inertial frame around the
  attractor. You can retrieve it using the {py:attr}`~poliastro.twobody.orbit.Orbit.frame` property:

    >>> orb.get_frame()
    <GCRS Frame (obstime=J2000.000, obsgeoloc=(0., 0., 0.) m, obsgeovel=(0., 0., 0.) m / s)>

### Intermezzo: quick visualization of the orbit

> <img align="right" width="350" alt="Plot of the orbit" src="_static/curtis.png">

Reviewer: Rather than using raw HTML, we can keep the

````
```{figure} ...
````

in this case! That's valid MyST :)

- we\'re visualizing the plane of the orbit itself,
- the $(x)$ axis points to the pericenter, and
- the $(y)$ axis is turned $90 \mathrm{^\circ}$ in the direction of the orbit.

The dotted line represents the *osculating orbit*: the instantaneous Keplerian orbit at that point. This is relevant in the context of perturbations, when the object shall deviate from its Keplerian orbit.

> > This visualization uses Plotly under the hood and works best in a Jupyter notebook.

Reviewer: We can use

````
```{note}
...
```
````

instead

man = Maneuver((0 * u.s, dv))  # Equivalent
```

> There are other useful methods you can use to compute common in-plane maneuvers, notably {py:meth} `~poliastro.maneuver.Maneuver.hohmann` and {py:meth}`~poliastro.maneuver.Maneuver.bielliptic` for `Hohmann`_ and `bielliptic`_ transfers respectively. Both return the corresponding ``Maneuver`` object, which in turn you can use to calculate the total cost

Reviewer: I wonder if this

```
`Hohmann`_
```

syntax works? 🤔 Or is it reStructuredText? I think there is another way of doing this in MyST

>>> tuple(val.decompose([u.km, u.s]) for val in hoh[1])
(<Quantity 15729.741535747102 s>, <Quantity [ 0.        , 1.41999995, 0.        ] km / s>)
```

> * [Hohmann](https://en.wikipedia.org/wiki/Hohmann_transfer_orbit)

Reviewer: Rather than including a list of links, these `.. _Hohmann` elements are "invisible" references that are used above. See for example https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links or https://docutils.sourceforge.io/docs/user/rst/quickref.html#external-hyperlink-targets

Therefore, we should keep them "invisible". Perhaps just move these to the "There are other useful methods..." paragraph above?

>>> tuple(val.decompose([u.km, u.s]) for val in hoh[1])
(<Quantity 15729.741535747102 s>, <Quantity [ 0.        , 1.41999995, 0.        ] km / s>)
```

To actually retrieve the resulting ``Orbit`` after performing a maneuver, use
the method {py:meth}`~poliastro.twobody.orbit.Orbit.apply_maneuver`:

```python
>>> ss_f = ss_i.apply_maneuver(hoh)
>>> ss_f
36000 x 36000 km x 0.0 deg (GCRS) orbit around Earth (♁)
```

> #### More advanced plotting: {py:class}`~poliastro.plotting.OrbitPlotter` objects

Reviewer: This should be a level 3 header, just like the previous one. Also, I realized the `OrbitPlotter` doesn't exist.

```suggestion
### More advanced plotting: `OrbitPlotter*` objects
```

> We previously saw the :py:func:`poliastro.plotting.plot` function to easily plot orbits. Now we'd like to plot several orbits in one graph (for example, the maneuver we computed in the previous section). For this purpose, we have {py:class}`~poliastro.plotting.OrbitPlotter` objects in the {py:mod}`~poliastro.plotting` module.

Reviewer: Just realized that this line is outdated, and we could change a few more things:

```suggestion
We previously saw the {py:meth}`~poliastro.twobody.Orbit.plot` method to easily plot orbits. Now we'd like to plot several orbits in one graph (for example, the maneuver we computed in the previous section). For this purpose, we have `OrbitPlotter*` objects in the {py:mod}`~poliastro.plotting` module.
```

Which produces this beautiful plot:

> <img align="center" width="350" alt="Plot of a Hohmann transfer" src="_static/hohmann.png">

Reviewer: As above, we can keep this `{figure}` directive

##### Where are the planets? Computing ephemerides

> versionadded: [0.14.0](https://github.com/poliastro/poliastro/tree/v0.14.0)

Reviewer: I have no idea how this is done in MyST. Could we perhaps leave `{eval-rst}` for this case? And/or open an issue on https://github.com/executablebooks/MyST-Parser?

---   
Plot of a Hohmann transfer.
```

> ##### Where are the planets? Computing ephemerides

Reviewer: This should also be a level 3 header:

```suggestion
### Where are the planets? Computing ephemerides
```

This does not require any external download. If on the other hand we
want to use higher precision ephemerides, we can tell Astropy to do so:

```shell
>>> from astropy.coordinates import solar_system_ephemeris
>>> solar_system_ephemeris.set("jpl")
Downloading http://naif.jpl.nasa.gov/pub/naif/generic_kernels/spk/planets/de430.bsp
|==========>-------------------------------|  23M/119M (19.54%) ETA    59s22ss23
```

This in turn will download the ephemerides files from NASA and use them for future computations. For more information, check out [Astropy documentation on ephemerides](https://docs.astropy.org/en/stable/coordinates/solarsystem.html).

> > :warning: ⚠ This is the preferred method over {py:meth}`poliastro.twobody.orbit.Orbit.from_body_ephem`, which is now deprecated and will be removed in the next release.

Reviewer: As above, we can use

````
```{warning}
...
```
````

(<Quantity 21910501.00019529 s>, <Quantity [287832.91384349,  58935.96079319, -94156.93383463] km / s>)
```

> <img align="center" alt="MSL orbit" src=" _static/msl.png">

Reviewer: As above, we can keep this `{figure}`
